delta/utils/register.py

```diff
@@ -152,16 +154,31 @@ def _handle_errors(errors):
     logging.fatal("Module {} import failed: {}".format(name, err))
 
 
+def path_to_module_format(py_path):
+  """Transform a python file path to module format."""
+  return py_path.replace("/", ".").rstrip(".py")
+
+
 def import_all_modules_for_register(config=None):
   """Import all modules for register."""
+  current_word_dir = os.getcwd()
+  if current_word_dir not in sys.path:
+    sys.path.append(current_word_dir)
   all_modules = ALL_SYS_MODULES
   errors = []
   if config is not None and "custom_modules" in config:
```

Comment:
what this for ?

Reply:
This is for customer module import searching.

Reply:
how to using this?